IPIP_PROCESS.md

```diff
@@ -0,0 +1,98 @@
+
+1. Provides good visibility into to the full set of proposals
+2. Keeps the full discussion for a proposal in one place, providing historical context
+3. Ensures stakeholders in the project can be aware of proposed changes and participate
+in the decision making process
+4. Provides a mechanism to ensure proposals are given consideration and decisions get made
+
+## Process design
+
+We adopted an formal change management process for the [ipfs/specs][1] repository, providing a
+minimal structure for opening, reviewing, and merging specification changes.
+
+### What is an IPIP?
+
+IPIP Provides an orderly mechanism for considering proposed changes to IPFS specifications.
+**An IPIP proposal is not to be the spec itself; the approval of an IPIP leads to an update to
+a specification.**
+
+To illustrate:
+- In order to understand how (hypothetical) WebDAV Gateway works, one would
+  read contents of specs in `ipfs/specs/WEBDAV_GATEWAY.md`.
+- IPIP in `ipfs/specs/IPIP/000N-webdav-gateway.md` would only include
+  **Motivation** and explainer why certain design decisions were made at a
+  certain point in time. Initial `IPIP/000N-webdav-gateway.md` would explain
+  why we added WebDAV spec in the first place.
+
+### What changes need the IPIP process?
+- **Does Not Need IPIP**: The spec has a bug - something that is plainly a mistake
+- **Does Not need IPIP**: Adding more details, test vectors, and editorials/cosmetic changes
+- **Needs IPIP**: An addition to the protocol
+- **Needs IPIP**:Things that could cause an interop issues require a PR with fix and IPIP in
+  `ipfs/specs/IPIP/000M-webdav-fix-for-foo.md` explaining why we make the
+  breaking spec change, compatibility/migration considerations etc.
+
+## Improvement lifecycle
+
+### Opening an improvement proposal (IPIP)
+
+Changes to IPFS specifications can be proposed by opening a Git pull-request
+(PR) against the `ipfs/specs` repository.
+
+In addition to specification changes, such PR must include a short **IPIP
+document** based on the template in [`ipfs/specs/IPIP/0000-template.md`](./0000-template.md).
+
+When a new specification file is added to the repo, it should be based on
+the template at [`ipfs/specs/template.md`](../template.md).
+
+When naming a new proposal, don't try to introduce an IPIP number; we will do that only for
+IPIPs that are approved before accepting into `main` branch.
+
+Proposals are officially submitted when a pull request into `main` is opened
+
+Proposals that were reviewed and rejected will be moved into `IPIP/rejected` folder and then merged into `main`
+
+### Reviewing IPIPs
+
+1. [Specs Stewards] will do an initial triage of newly opened PRs roughly monthly. They'll try to filter out
+noise, so that community consideration is given only to reasonable proposals; others they'll reject.
+2. Specs Stewards will post to the forums linking to the proposal; directing feedback/discussion to
```

Comment:
What do we mean by "the forums"?
Should I create "IPIP" category under https://discuss.ipfs.tech/c/protocol/30 ?